File: sqlparser/src/sql/SqlConditionNode.java
//@
package cdata.sql;
//@
/*#
using RSSBus.core;
using RSSBus;
namespace CData.Sql {
#*/

abstract public class SqlConditionNode extends SqlExpression implements ISqlElement {
  public abstract boolean isLeaf();
  public abstract void prepare() throws Exception;
  public abstract void accept(ISqlQueryVisitor visitor) throws Exception;

  @Override
  protected void copy(SqlExpression obj) {
  }
}

File: sqlparser/src/sql/csgen/SqlConditionNode.cs
using System;
using System.Collections;
using System.IO;
using System.Threading;
using System.Net;
using System.Net.Sockets;
using System.Text;
using RSSBus.core.j2cs;



using RSSBus.core;
using RSSBus;
namespace CData.Sql {


abstract public class SqlConditionNode : SqlExpression , ISqlElement {
  public abstract bool IsLeaf();
  public abstract void Prepare() ;
  public abstract void Accept(ISqlQueryVisitor visitor) ;

  protected override void Copy(SqlExpression obj) {
  }
}
}

